.github/workflows/build.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: sources
-          key: build-sources-${{ github.run_id }}
+          key: build-sources-${{ github.run_id }}-${{ github.run_number }}
```

Comment:
Should we modify the same file in apps and testing git?

Reply:
We need to just drop the cache.  We always miss no that GitHub has made cache faster.